docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.3'
+services:
+  xu:
+    image: exchangeunion
+    build:
+      context: .
+    volumes:
+      - ./:/opt/xud
+    environment:
+      DB_HOST: "db"
+      DB_PORT: 3306
+      DB_USER: "root"
```

Comment:
Is running as root alright here? I'm not sure about best practices for docker containers.

Reply:
Well, the docker-compose file currently is only for dev environment. We need a init script to create other non-root users and database.